hugo/content/uninstall.md

```diff
@@ -0,0 +1,145 @@
+---
+title: "Uninstall NGINX Agent Package"
+draft: false
+description: "Learn how to uninstall NGINX Agent"
+weight: 500
+toc: true
+tags: [ "docs" ]
+docs: "DOCS-1219"
+categories: ["configuration"]
+doctypes: ["task"]
+---
+
+## Prerequisites
+
+- NGINX Agent installed
+- `root` privilege
```

Comment:
Please provide more details and link to the relevant documentation

Reply:
Provided link to NGINX Agent install and provide details the user requires to follow provided steps